### PR TITLE
Code Quality Improvements for Application-mgt Module

### PR DESCRIPTION
**Pull request by kasungayan**
- Removing plausible Critical, Major and Minor issues related to Application-mgt Module based on SonarQube results.
- Updating Licence Headers.

### CODE REVIEW

File: components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/IdentityApplicationManagementUtil.java [hunk 1]
```diff
@@ -909,7 +918,7 @@ public static String getSignedJWT(String jsonObj, ServiceProvider serviceProvide
                     .getInboundAuthenticationConfig().getInboundAuthenticationRequestConfigs();
 
             for (InboundAuthenticationRequestConfig authReqConfig : authReqConfigs) {
-                if (authReqConfig.getInboundAuthType().equals("oauth2")) {
+                if ("oauth2".equals(authReqConfig.getInboundAuthType())) {
```
Comment (prabathabey) [inline components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/IdentityApplicationManagementUtil.java]: Use appropriate constants for terms like "oauth2", etc.

File: components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/IdentityApplicationManagementUtil.java [hunk 2]
```diff
@@ -970,6 +979,9 @@ public static String calculateHmacSha1(String key, String value) throws Signatur
             byte[] rawHmac = mac.doFinal(value.getBytes());
             result = Base64Utils.encode(rawHmac);
         } catch (Exception e) {
+            if(log.isDebugEnabled()){
```
Comment (prabathabey) [inline components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/IdentityApplicationManagementUtil.java]: Formatting issues. Please fix.

File: components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/IdentityApplicationManagementUtil.java [hunk 3]
```diff
@@ -970,6 +979,9 @@ public static String calculateHmacSha1(String key, String value) throws Signatur
...
+                log.debug("Failed to create the HMAC Signature",e);
```
Comment (prabathabey) [inline components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/IdentityApplicationManagementUtil.java]: More formatting issues.

File: components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/IdentityApplicationManagementUtil.java (hunk 1 above)
Comment (kasungayan) [inline components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/IdentityApplicationManagementUtil.java]: Fixed with #1545

File: components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/IdentityApplicationManagementUtil.java (hunk 2 above)
Comment (kasungayan) [inline components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/IdentityApplicationManagementUtil.java]: Fixed with #1545

File: components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/IdentityApplicationManagementUtil.java (hunk 3 above)
Comment (kasungayan) [inline components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/util/IdentityApplicationManagementUtil.java]: Fixed with #1545

**Pull request merged**